packages/salesforcedx-utils-vscode/src/context/workspaceContextUtil.ts

```diff
@@ -8,7 +8,8 @@
 import { AuthInfo, Connection } from '@salesforce/core';
 import { join } from 'path';
 import * as vscode from 'vscode';
-import { AuthUtil, ConfigAggregatorProvider } from '..';
+import { ConfigAggregatorProvider } from '..';
+import { AuthUtil } from '../auth/authUtil';
```

Comment:
This was required in order to not mock the entirety of the utils-vcode module. I think our pattern should be for internal imports (same package)  in code we import directly from the file where exported.  For importing from other packages use the top level export.